Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -109,10 +117,14 @@ def genGridOnPoints(data1,gm,deep=True,grid=None,geo=None):
     x=numpy.reshape(x,sh)
     y=numpy.reshape(y,sh)
     #Ok we have our points in 2D let's create unstructured points grid
-    xm=x.min()
-    xM=x.max()
-    ym=y.min()
-    yM=y.max()
+    if xm is None:
+        xm=x.min()
+    if xM is None:
+        xM=x.max()
+    if ym is None:
+        ym=y.min()
```

Comment:
minor formatting issues.. spacing before and after = is inconsitent. No worries. we will have a style test.